Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -36,6 +36,15 @@ static bool Init()
             return LoadConfig();
         }
 
+        /// <summary>
+        /// Manually initialize the SDK.
+        /// </summary>
+        /// <param name="apiKey">Find the Game API-key at https://my.lootlocker.io/settings/game and click on the API-tab</param>
+        /// <param name="gameVersion">The current version of the game in the format 1.2.3.4 (the 3 and 4 being optional but recommended)</param>
```

Comment:
Refer to semver documentation instead like: "The current version of the game in semver format [link]" ?

Reply:
I don't agree. It's not necessary to know the format of the API-key that you are pasting in, you just need to knów where to find it. It will just be more information for the player that they don't need to know about.

Reply:
Okay, I guess I see what you mean. But to me it's different as sure; the api key is just something they get from us and need to paste. But the version is something they set themselves and if we enforce Semver then that should be mentioned. Not that important regardless.